root/static/styles/layout.less

```diff
@@ -1015,6 +1015,11 @@ a.tagger-icon {
     min-width: 10em;
```

Comment:
I'd think you can remove this rule now, and the `actions-header` class with it, since the column will never wrap now (it was added for [MBS-5395](https://tickets.metabrainz.org/browse/MBS-5395), but having `white-space: nowrap` resolves that just as well).